src/Razor/src/Microsoft.AspNetCore.Razor.VSCode/src/ConfigurationChangeListener.ts

```diff
@@ -0,0 +1,32 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+import * as vscode from 'vscode';
+import { RazorLanguageServerClient } from './RazorLanguageServerClient';
+
+export function listenToConfigurationChanges(
+    languageServerClient: RazorLanguageServerClient) {
+    vscode.workspace.onDidChangeConfiguration(event => {
```

Comment:
This returns an `vscode.Disposable` that needs to be added to our extension contexts subscription list [like we do with other registrations](https://github.com/dotnet/aspnetcore-tooling/blob/d12492edf118463b5de8e091cccce1d1827eb10c/src/Razor/src/Microsoft.AspNetCore.Razor.VSCode/src/extension.ts#L129-L165).

Reply:
Got it, thanks!